pkg/io/size_test.go

```diff
@@ -91,7 +90,7 @@ func TestVarSize(t *testing.T) {
 		},
 		{
 			// The neo C# implementation doe not allowed this!
-			util.Uint160{1, 2, 4, 5, 6},
+			[20]byte{1, 2, 4, 5, 6},
```

Comment:
Hmm?

Reply:
There was a circular dependency between `io` and `util` only because of this test (and this prohibits implementation of `io.Serializable` for any types in `util` package).
If this tests are needed may be it is better to move them into `util` package?

Reply:
I have renamed this package to `size_test` instead, luckily it uses no private `size`'s variables.